x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -264,19 +269,21 @@ export const getManagementFilteredLinks = async (
       linksToExclude.push(SecurityPageName.responseActionsHistory);
     }
 
-    if (!canIsolateHost && canUnIsolateHost) {
-      let shouldSeeHIEToBeAbleToDeleteEntries: boolean;
+    if (canIsolateHost && !hasPlatinumLicense) {
```

Comment:
Knowning that you are really using `canUnIsolateHost` here (due to rename), I wonder if this is really correct. I think (if I remember correctly) that `un-isolate` can actually be available in cases where the user downgraded and a host might still be isolate (they should be able to un-isolate in that case).

Reply:
Quite right about the `canUnIsolate` here. Again, all we need here is `canIsolateHost` that authz exposes which is a conjunction of `canIsolateHost` and `platinum` license checks.

Reply:
undid the rename for permissions 8130caa06366943913c0c4bbd8c7a1c6970c404e